### PR TITLE
[5.6] Add words to uncountable list

### PR DESCRIPTION
**Pull request by tnorthcutt**

### CODE REVIEW

**Comment by taylorotwell** [comment]
Not so sure about "litter"... "3 litters of cats".

**Comment by GrahamCampbell** [comment]
I kinda feel like we are going overboard with this stuff. Is there some other package we can depend on that'll do this?

**Comment by ntzm** [comment]
It should be dealt with by inflector? https://github.com/doctrine/inflector/blob/master/lib/Doctrine/Common/Inflector/Inflector.php#L227-L240

File: src/Illuminate/Support/Pluralizer.php [hunk 1]
```diff
@@ -45,12 +48,14 @@ class Pluralizer
...
         'rain',
         'rice',
+        'sand',
         'series',
         'sheep',
         'software',
         'species',
         'swine',
         'traffic',
+        'water',
```
Comment (ntzm) [inline src/Illuminate/Support/Pluralizer.php]: Water is countable

File: src/Illuminate/Support/Pluralizer.php [hunk 2]
```diff
@@ -45,12 +48,14 @@ class Pluralizer
         'police',
         'rain',
         'rice',
+        'sand',
```
Comment (ntzm) [inline src/Illuminate/Support/Pluralizer.php]: Sand is countable:

> The Canadian tar sands are a promising source of oil.

File: src/Illuminate/Support/Pluralizer.php [hunk 3]
```diff
@@ -33,8 +34,10 @@ class Pluralizer
         'jedi',
         'kin',
         'knowledge',
+        'litter',
         'love',
         'metadata',
+        'milk',
```
Comment (ntzm) [inline src/Illuminate/Support/Pluralizer.php]: Milk is countable:

> Table three ordered three milks. (Formally: The guests at table three ordered three glasses of milk.)

File: src/Illuminate/Support/Pluralizer.php [hunk 4]
```diff
@@ -20,6 +20,7 @@ class Pluralizer
         'data',
         'deer',
         'education',
+        'elk',
```
Comment (ntzm) [inline src/Illuminate/Support/Pluralizer.php]: I believe doctrine/inflector should already deal with this

**Comment by BrandonShar** [comment]
I wish I worked on fun enough apps to run into issues with my `Elk`, `Litter`, and `Sand` models. :smile:

**Pull request closed without merging**